docs/content/Getting-Started/Cloud/02-Create-a-deployment.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Create a deployment
+permalink: /getting-started/cloud/create-a-deployment
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 3
+isDisableFeedbackBlock: true
+---
+
+## Create deployment
+
+The first step in creating a deployment is to sign in to your Cube Cloud
+account. After signing in, click the **Create Deployment** button:
```

Comment:
How exactly would one sign up? I think it would be great to add a link to the sign-up page